readme.md

```diff
@@ -60,11 +58,11 @@ curl -X POST \
   http://localhost:4005/api/v1
 ```
 
-but see https://github.com/statechannels/nitro-gui for an RPC client to do so programmatically.
+but see the [Go rpc client](./rpc/client.go) or the [Typescript rpc client](https://github.com/statechannels/nitro-gui) for an RPC client to do so programmatically.
```

Comment:
Suggestion: change link to the `nitro-rpc-client` package within `nitro-gui` repo: https://github.com/statechannels/nitro-gui/tree/main/packages/nitro-rpc-client